distribution/src/main/gateway/bin/gateway.start

```diff
@@ -56,5 +56,17 @@ fi
 # Add a directory for management support
 JAVA_LIBRARY_PATH="$GW_HOME/lib/sigar"
 
-exec java $GATEWAY_OPTS -Djava.library.path="$JAVA_LIBRARY_PATH" -XX:+HeapDumpOnOutOfMemoryError -cp "$GW_HOME/lib/*" org.kaazing.gateway.server.Main $*
+GW_ID=""
 
+# Verify if the gateway identifier was provided (required by Agorna multiple gateway instances)
+if [[ $GATEWAY_OPTS == *"org.kaazing.gateway.management.AGRONA_ENABLED=true"* ]]; then
+  if [ "$GATEWAY_IDENTIFIER" != "" ]; then
+    GW_ID="-Dorg.kaazing.gateway.server.GATEWAY_IDENTIFIER=$GATEWAY_IDENTIFIER"
+
+    # Startup the StatsD publisher
+    ./scripts/metrics.statsD.start $GATEWAY_IDENTIFIER &
```

Comment:
We can call ../metrics.statsd/metrics.statsd.start directly if we alter that script so it can be called from anywhere. I have proposed a change to allow that in my metrics pull request https://github.com/kaazing/metrics/pull/8